reconcile/vault_replication.py

```diff
@@ -325,6 +344,19 @@ def replicate_paths(
             for vault_path in path_list:
                 copy_vault_secret(dry_run, source_vault, dest_vault, vault_path)
 
+        elif isinstance(path, VaultReplicationPolicyV1):
+            vault_query_data = vault_policies.query(query_func=gql.get_api().query)
+            if path.policy is None:
+                raise VaultInvalidPolicy
```

Comment:
Thanks for the comment. I was meaning more an exception message so that it is easy to identify what's going on and even identify it in the code in case you reuse that exception.